packages/expo-updates/ios/EXUpdates/EXUpdatesDevExtension.m

```diff
@@ -0,0 +1,53 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import "EXUpdatesDevExtension.h"
+
+@implementation EXUpdatesDevExtension
+
+RCT_EXPORT_MODULE(EXUpdatesDevExtension)
+
+RCT_EXPORT_METHOD(getUpdatesConfigAsync:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+  NSMutableDictionary *updatesConfig = [NSMutableDictionary new];
+  
+  NSString *runtimeVersion = [self getUpdatesConfigForKey:@"EXUpdatesRuntimeVersion"];
+  NSString *sdkVersion = [self getUpdatesConfigForKey:@"EXUpdatesSDKVersion"];
+  
+  // url structure for EASUpdates: `http://u.expo.dev/{appId}`
+  // this url field is added to app.json.updates when running `eas update:configure`
+  // the `u.expo.dev` determines that it is the modern manifest protocol
+  NSString *updatesUrl = [self getUpdatesConfigForKey:@"EXUpdatesURL"];
+  NSURL *url = [NSURL URLWithString:updatesUrl];
+  NSString *appId = [[url pathComponents] lastObject];
+  
+  BOOL isModernManifestProtocol = [[url host] isEqualToString:@"u.expo.dev"];
+  BOOL isEASUpdatesInstalled = isModernManifestProtocol && appId.length > 0;
+  
+  [updatesConfig setObject:runtimeVersion forKey:@"runtimeVersion"];
+  [updatesConfig setObject:sdkVersion forKey:@"sdkVersion"];
+  [updatesConfig setObject:appId forKey:@"appId"];
+  [updatesConfig setObject:@(isEASUpdatesInstalled) forKey:@"isEASUpdatesInstalled"];
```

Comment:
something like `usesEASUpdate` might be a more accurate name for this field, since there isn't much that gets "installed" when you configure EAS Update